### PR TITLE
Add VALIDATE_EMPTY_FLOOR chartValidation key

### PR DESCRIPTION
**Pull request by nahue**

### CODE REVIEW

File: src/Accounts/ChartValidationSettings.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ class ChartValidationSettings {
         this.VALIDATE_UNLABELED_OBJECTS = chartValidations.VALIDATE_UNLABELED_OBJECTS
         this.VALIDATE_FOCAL_POINT = chartValidations.VALIDATE_FOCAL_POINT
         this.VALIDATE_OBJECT_TYPES_PER_CATEGORY = chartValidations.VALIDATE_OBJECT_TYPES_PER_CATEGORY
+        this.VALIDATE_EMPTY_FLOOR = chartValidations.VALIDATE_EMPTY_FLOOR
```
Comment (mroloux) [inline src/Accounts/ChartValidationSettings.js]: Looks like we check for the presence of these keys in `retrieveMyAccount.test.js`

Comment (nahue) [reply src/Accounts/ChartValidationSettings.js]: Added!

Comment (nahue) [reply src/Accounts/ChartValidationSettings.js]: Keep in mind that the test will fail because staging is not updated

Comment (mroloux) [reply src/Accounts/ChartValidationSettings.js]: 👍

**Pull request merged**